src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -90,35 +91,123 @@ public void TestInitialize()
+            act = () => new BindingProcessImpl(validHost, bindingArgs, slnBindOp, null, finder, bindingConfigProvider, SonarLintMode.Connected, exclusionSettingsStorage);
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("nugetBindingOperation");
 
             // 5. Null binding info provider
-            act = () => new BindingProcessImpl(validHost, bindingArgs, slnBindOp, nuGetOp, null, bindingConfigProvider, SonarLintMode.Connected);
+            act = () => new BindingProcessImpl(validHost, bindingArgs, slnBindOp, nuGetOp, null, bindingConfigProvider, SonarLintMode.Connected, exclusionSettingsStorage);
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("unboundProjectFinder");
 
             // 6. Null rules configuration provider
-            act = () => new BindingProcessImpl(validHost, bindingArgs, slnBindOp, nuGetOp, finder, null, SonarLintMode.Connected);
+            act = () => new BindingProcessImpl(validHost, bindingArgs, slnBindOp, nuGetOp, finder, null, SonarLintMode.Connected, exclusionSettingsStorage);
             act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("bindingConfigProvider");
+
+            // 6. Null exclusion settings storage
+            act = () => new BindingProcessImpl(validHost, bindingArgs, slnBindOp, nuGetOp, finder, bindingConfigProvider, SonarLintMode.Connected, null);
+            act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("exclusionSettingsStorage");
+        }
+
+        [TestMethod]
+        public async Task SaveServerExclusionsAsync_ConnectedMode_ReturnsTrue()
+        {
+            var bindingArgs = CreateBindCommandArgs(projectKey: "projectKey");
+
+            CreateConfigurationProviderService("C:\\SolutionPath");
+
+            ServerExclusions settings = CreateSettings();
+
+            var sonarQubeService = new Mock<ISonarQubeService>();
+            sonarQubeService.Setup(s => s.GetServerExclusions("projectKey", It.IsAny<CancellationToken>())).Returns(Task.FromResult(settings));
+
+            var exclusionSettingsStorage = new Mock<IExclusionSettingsStorage>();
+
+            var testSubject = CreateTestSubject(bindingArgs: bindingArgs, mode: SonarLintMode.Connected, sonarQubeService: sonarQubeService.Object, exclusionSettingsStorage: exclusionSettingsStorage.Object);
+            
+            await testSubject.SaveServerExclusionsAsync(CancellationToken.None);
+
+            exclusionSettingsStorage.Verify(fs => fs.SaveSettings(settings), Times.Once);
+            this.outputWindowPane.AssertOutputStrings(0);
+        }
+
+        [TestMethod]
+        public async Task SaveServerExclusionsAsync_HasError_ReturnsFalse()
+        {
+            var bindingArgs = CreateBindCommandArgs(projectKey: "projectKey");
```

Comment:
Can this be a default param in CreateTestSubject?

Reply:
It's a default parameter used in other tests already. I don't want to change it because I wouldn't like it to affect existing tests. So I need to create it in new tests.